drivers/gps/nrf9160_gps/Kconfig

```diff
@@ -38,6 +38,24 @@ config NRF9160_GPS_HANDLE_MODEM_CONFIGURATION
 	  application send the AT commands to configure the GPS
 	  properly.
 
+config NRF9160_GPS_PRIORITY_WINDOW_TIMEOUT_SEC
+	int "Enable GPS priority window if the GPS is blocked for this duration"
+	default 10
+	help
+	  If the GPS is blocked for more than the amount of time specified by
+	  this option, and the GPS priority window is enabled in the passed in
+	  GPS configuration structure, the GPS priority window is requested by
+	  the driver. When the GPS priority window is enabled, the modem
+	  prioritizes a fix over LTE activity. If the value of this option is
```

Comment:
Perhaps something like ` the modem grants the GPS prioritized access to the radio at the cost of LTE activity`

Reply:
Yes, that has a better ring to it.